targets/TARGET_WIZNET/TARGET_W7500x/analogin_api.c

```diff
@@ -58,11 +58,11 @@ void analogin_init(analogin_t *obj, PinName pin)
     obj->pin = pin;
 
     // The ADC initialization is done once
-    if (adc_inited == 0) {
+    //if (adc_inited == 0) {
```

Comment:
second ADC pin will reinit entire ADC? What is this change fixing? What was wrong with the previous version?

For future reference, It would be beneficial if any logical change would have separate commit, like this one, would be explained via commit message why and what has changed.